tests/bal_business_logic_suite.py

```diff
@@ -73,38 +147,22 @@ class Test_managementPolicy_library_specified_behavior(FixtureAugmentedTestCase)
         {"a", "managed", "trait", "set"},
     )
 
-    # We need to load a different library for these tests. This is
-    # a little fiddly as the manager isn't (currently) recreated
-    # for each test.
-
-    ## @todo Extend openassetio.test.manager to allow settings to be
-    ## varied per test class/method.
-
-    def setUp(self):
-        self.__old_settings = self._manager.settings()
-        new_settings = self.__old_settings.copy()
-        new_settings["library_path"] = os.path.join(
-            os.path.dirname(__file__),
-            "resources",
-            "library_business_logic_suite_customManagementPolicy.json",
-        )
-        self._manager.initialize(new_settings)
-
-    def tearDown(self):
-        self._manager.initialize(self.__old_settings)
-
     def test_returns_expected_policies_for_all_trait_sets(self):
         context = self.createTestContext(access=Context.Access.kRead)
-        expected = [TraitsData({ManagedTrait.kId}), TraitsData(), TraitsData({ManagedTrait.kId})]
-        ManagedTrait(expected[0]).setExclusive(True)
+        expected = [
+            TraitsData({"openassetio.Managed"}),
+            TraitsData(),
+            TraitsData({"openassetio.Managed"}),
+        ]
+        expected[0].setTraitProperty("openassetio.Managed", "exclusive", True)
 
         actual = self._manager.managementPolicy(self.__read_trait_sets, context)
 
         self.assertListEqual(actual, expected)
 
     def test_returns_expected_policy_for_write_for_all_trait_sets(self):
         context = self.createTestContext(access=Context.Access.kWrite)
-        expected = [TraitsData(), TraitsData({ManagedTrait.kId})]
+        expected = [TraitsData(), TraitsData({"openassetio.Managed"})]
```

Comment:
[consider] Is it worth making this contrived, to avoid any confusion with real traits that do exist?

Reply:
Good shout, updated to something more obviously just for testing in 7dca6b6c.  

I made it conform to traitgen syntax to make it more representative, though. Not sure that was a good idea, thinking about it, since it kinda undoes some of the contrivedness.

Reply:
I think that's good enough though - at least someone reading that won't think that's a valid trait for normal use.